packages/android-ndk/r14b/version.desc

```diff
@@ -0,0 +1 @@
+obsolete='yes'
```

Comment:
You can put this in package.desc, that will declare all versions obsolete (bootstrap first applies settings from package.desc then overrides them from a particular version.desc).

Reply:
Done.